pkg/pillar/types/ifnametopci.go

```diff
@@ -160,8 +161,8 @@ func IoBundleToPci(ib *IoBundle) (string, error) {
 		return "", nil
 	}
 	if !pciLongExists(long) {
-		errStr := fmt.Sprintf("PCI device name %s id %s does not exist",
-			ib.Name, long)
+		errStr := fmt.Sprintf("PCI device name %s ifname %s id %s does not exist",
```

Comment:
This change is incorrect. We do PCI lookups for things like audio and USB controller, which do not have an ifname nor should they.

Reply:
Then it would be an empty string.. isn't it? This is harmless in that case.. right?